client/blocks/login/error-notice.jsx

```diff
@@ -57,7 +57,7 @@ class ErrorNotice extends Component {
 	render() {
 		const error = this.getError();
 
-		if ( ! error || ( error.field && error.field !== 'global' ) || ! error.message ) {
+		if ( ! error || ( error.field && error.field !== 'global' ) || ! error.message || error.code === 'existing_wpcom_user' ) {
```

Comment:
This is quite a long condition. Maybe it would be nice to move it to a function?